test/integration/scripts/fast_test

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash -x
+
+# Copyright 2019 The KubeEdge Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cd `dirname $0`
+workdir=`pwd`
+cd $workdir
+
+debugflag="-v 6 -alsologtostderr"
+modulename=$1
+
+#setup env
+cd ../
+#Pre-configurations required for running the suite.
+#Any new config addition required corresponding code changes.
+cat >config.json<<END
+{
+        "mqttEndpoint":"tcp://$MQTT_SERVER:1884",
+        "deviceId": "1234567890",
```

Comment:
I am not sure why do we need to have a hard-coded device id ? can't we generate device uuid's programatically ?

Reply:
@rohitsardesai83 , Removed from configuration. generated deviceid with random string programatically.